src/index.html

```diff
@@ -2,11 +2,407 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+    ">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <div class="header_content">
+        <a class="logo" href="#home">
+          <img
+            class="logoImg"
+            src="images/logo.png"
+            alt="Moyo logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class = "nav__links is-active" href="#apple">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a class = "nav__links" href="#samsung">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a class = "nav__links" href="#smart_and_tel">Смартфоны и телефоны</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class = "nav__links"
+                href="#laptop_and_pc"
+                data-qa="nav-hover"
+              >Ноутбуки и компьютеры
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a class = "nav__links" href="#gadget">Гаджеты</a>
+            </li>
+
+            <li class="nav__item">
+              <a class = "nav__links" href="#tab">Планшеты</a>
+            </li>
+
+            <li class="nav__item">
+              <a class = "nav__links" href="#photo">Фото</a>
+            </li>
+
+            <li  class="nav__item">
+              <a class = "nav__links" href="#video">Видео</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="content">
+      <div class="content__contaiter">
+        <div class="content__item">
+          <div class="product" data-qa="card">
```

Comment:
```suggestion
    <main class="content">
      <div class="content__contaiter">
        <div class="content__item">
          <div class="product" data-qa="card">
```
Лучше прописать классы по BEM.
И, может, избавиться от лишней обёртки, чтобы использовать немного меньше кода?

Reply:
Agree